bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcUniqueConfig.scala

```diff
@@ -92,13 +92,19 @@ object ItcUniqueConfig {
   // Creates a text label based on the spans of steps covered by the steps.
   // E.g. ((1,2,3),(10,11),(15)) is turned into "001-003, 010-011, 015"
   private def labels(cs: Seq[Config]): String = {
-    val ls = cs.map(_.getItemValue(DATALABEL_KEY).asInstanceOf[String].toInt).sorted
+    val ls = cs.map(labelIndex).sorted
     findSpans(ls).map {
       case i :: Nil => i.toString
       case i :: is => f"$i%03d-${is.last}%03d"
     }.mkString(", ")
   }
 
+  // Only the number at the very end is relevant
+  private def labelIndex(c: Config): Int = {
+    val label = c.getItemValue(DATALABEL_KEY).asInstanceOf[String]
+    label.split('-').last.toInt
```

Comment:
Even for an empty string ```split()``` does not return an empty array but an array with an empty string:

```scala
scala> "".split('-')
res0: Array[String] = Array("")

scala> "123".split('-')
res1: Array[String] = Array(123)

scala> "123-123".split('-')
res2: Array[String] = Array(123, 123)
```